clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/callbacks/Utils.groovy

```diff
@@ -277,7 +277,7 @@ class Utils {
   static boolean determineInternalLoadBalancerDisabledState(GoogleInternalLoadBalancer loadBalancer,
                                                             GoogleServerGroup serverGroup) {
     def regionalLoadBalancersFromMetadata = serverGroup.asg.get(GoogleServerGroup.View.REGIONAL_LOAD_BALANCER_NAMES)
-    List<GoogleLoadBalancedBackend> serviceBackends = loadBalancer.backendService.backends
+    List<GoogleLoadBalancedBackend> serviceBackends = loadBalancer?.backendService?.backends
```

Comment:
Would be a good idea to log the fact that we encountered a malformed load balancer representation here and below.